components/tabs/tabs.js

```diff
@@ -573,38 +574,37 @@ class Tabs extends LocalizeStaticMixin(ArrowKeysMixin(RtlMixin(LitElement))) {
 		}
 		this._tabInfos = newTabInfos;
 
-		if (!this._initialized && this._tabInfos.length > 0) {
+		if (this._tabInfos.length > 0 && !selectedTabInfo) {
```

Comment:
Previously we only defaulted to select the first tab (i.e. when no other panel is selected) on first render.  This change moves that logic out, so that it will select the first tab in the even that the selected tab is removed and the consumer has not previously selected a different tab.